.buildkite/pipeline.yml

```
@@ -1,5 +1,118 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  GOLANG_VERSION: "1.21.3"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2004"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  DOCKER_REGISTRY_SECRET_PATH: "secret/observability-team/ci/docker-registry/prod"
+  BUILDX: 1
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - group: "Staging"
+    key: "staging"
+
+    steps:
+      - label: ":unix: Staging / Ubuntu X86_64 - {{matrix.makefile}}"
+        key: "build-ubuntu-x86"
+        command:
+          - ".buildkite/scripts/build.sh {{matrix.makefile}}"
+          - ".buildkite/scripts/publish.sh {{matrix.makefile}}"
+        if: build.env("BUILDKITE_PULL_REQUEST") != "false"
+        notify:
+          - github_commit_status:
+              context: "Staging / Ubuntu X86_64"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+        matrix:
+          setup:
+            makefile:
+              - "Makefile"
+              - "Makefile.debian7"
+              - "Makefile.debian8"
+              - "Makefile.debian9"
+              - "Makefile.debian10"
+              - "Makefile.debian11"
+              - "Makefile.debian12"
+
+      - label: ":unix: Staging / Ubuntu ARM - {{matrix.makefile}}"
```

unix is not part of the emoji list available in Buildkite
https://github.com/buildkite/emojis#emoji-reference

What about using :linux:? 
```suggestion
      - label: ":linux: Staging / Ubuntu ARM - {{matrix.makefile}}"
```

---

There are other usages of this in 
https://github.com/elastic/golang-crossbuild/pull/326/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53R68
https://github.com/elastic/golang-crossbuild/pull/326/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53R88